javascript/ql/src/LanguageFeatures/SyntaxError.ql

```diff
@@ -13,4 +13,5 @@
 import javascript
 
 from JSParseError pe
+where not pe.getTopLevel() instanceof Angular2::TemplateTopLevel
```

Comment:
What is the motivation for ignoring angular-templates for syntax-errors?

Reply:
The syntax errors [here](https://jenkins-artifacts.internal.semmle.com/static-files/eDSkVvRutOrltXAbthwpAerG30zLxrs1IKpA_nUJ3eoxNjExODQ0ODE1NzM0OjU6YXNnZXI6am9iL0NoYW5nZXMvam9iL0pTLURpZmZlcmVuY2VzLzM5Mi9hcnRpZmFjdA==/changes.html) (internal link).